src/pip/_internal/resolution/resolvelib/factory.py

```diff
@@ -105,6 +104,10 @@ def __init__(
         else:
             self._installed_dists = {}
 
+    @property
+    def force_reinstall(self):
```

Comment:
This needs an annotation, as per the lint check.

Reply:
Fixed. I wish Mypy is smart enough to infer this kind of simple annotations 😥